sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -231,7 +236,7 @@ protected String getLoggerTag() {
     public synchronized void onActivityResumed(Activity activity) {
         mActivity = activity;
         if (mSignInDelayed) {
-            signInFromUI();
+            signInInteractively();
```

Comment:
This is actually wrong as it does not try to use silent sign-in but we are going to remove delayed sign-in very soon so we can let this go I guess.

Reply:
Expecting fix for this in #982 that should remove delayed sign-in.